vertical-pod-autoscaler/pkg/target/v1beta1_fetcher.go

```diff
@@ -0,0 +1,87 @@
+/*
```

Comment:
This file looks unused - why is it part of the PR?

Reply:
Ah, thanks! Rebase confusion. Removed.